### PR TITLE
chore: rename contracts to add RootstockCollective postfix

### PR DESCRIPTION
**Pull request by TravellerOnTheRun**

### CODE REVIEW

File: contracts/test/GovernorV2RootstockCollective.sol [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Ini
 import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import {UUPSUpgradeable} from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 
-contract GovernorV2 is
+contract GovernorV2RootstockCollective is
```
Comment (raullaprida) [inline contracts/test/GovernorV2RootstockCollective.sol:16]: In __Governor_init("RootstockCollective");
I would change it to 
__Governor_init("GovernorV2RootstockCollective");
Just to be consistent, even though this is under the test path

**Review by raullaprida — COMMENTED** [review]
I just added one comment for GovernorV2.sol

Also, from the task, it was required to also change the name of Proxy.sol to ProxyRootstockCollective.sol (and the name of the contract to ProxyRootstockCollective).
This file was not changed, so I went to look at the code it contains.  I noticed it's just the OZ one, then why not just use the OZ one? Otherwise if we create a new .sol it will most likely require the branding required in this task (and probably just inherit the OZ implementation in a contract that has ProxyRootstockCollective as name)

**Review by cybervoid0 — APPROVED** [review]
All Good! I changed the Treasury interface name too

File: contracts/test/GovernorV2RootstockCollective.sol (hunk 1 above)
Comment (cybervoid0) [reply contracts/test/GovernorV2RootstockCollective.sol:16]: >  I noticed it's just the OZ one, then why not just use the OZ one?

I didn't got your question. We actually used the OZ one.
This file is needed by Hardhat to reference OZ proxy in order to generate artifacts. It won't work without it. But we didn't inherit from it, we just used proxy as is.

**Pull request merged**